src/database.c

```diff
@@ -493,3 +555,159 @@ _add_to_db(ProfMessage* message, char* type, const Jid* const from_jid, const Ji
         }
     }
 }
+
+static int
+_get_db_version(void)
+{
+    int current_version = -1;
+    const char* query = "SELECT `version` FROM `DbVersion` LIMIT 1";
+    sqlite3_stmt* statement;
+
+    if (sqlite3_prepare_v2(g_chatlog_database, query, -1, &statement, NULL) == SQLITE_OK) {
+        if (sqlite3_step(statement) == SQLITE_ROW) {
+            current_version = sqlite3_column_int(statement, 0);
+        }
+        sqlite3_finalize(statement);
+    }
+    return current_version;
+}
+
+/**
+ * @brief Migration to V2 that introduces new fields and reorganizes message storage.
+ * `replace_id` -> `replaces_stanza_id` stanza ID of the replaced message
+ * `replaces_db_id` db ID of the replaced message
+ * `original_message` original message before replacement
+ * `message` now is used to store visual message (replaced by LMC)
+ *
+ * @return gboolean with migration success status.
```

Comment:
Remove `@return gboolean with migration success status.` It doesnt tell us anything. It doesnt tell whether true is success or false so we know exactly the same like just reading the function header.
And as previously said in reviews we dont want doxygen style docu.